explorer/ExplorerProgram.test.ts

```diff
@@ -462,4 +462,47 @@ france,Life expectancy`
             expect(cell.errorMessage).not.toEqual(``)
         })
     })
+
+    describe("defaultView", () => {
+        const matrix = `${grapherIdKeyword},Metric Dropdown,Interval Dropdown,Relative to population Checkbox,defaultView
+1,Cases,Daily,true
+2,Cases,Weekly,true
+3,Cases,Cumulative,true
+4,Cases,Cumulative,false
+5,Tests,Cumulative,true,true
+6,Tests,Cumulative,false`
+
+        it("can set a default view", () => {
+            const decisionMatrix = new DecisionMatrix(matrix)
+
+            expect(decisionMatrix.selectedRow.grapherId).toEqual(5)
+        })
+
+        it("can override defaults", () => {
+            const decisionMatrix1 = new DecisionMatrix(matrix)
+
+            decisionMatrix1.setValuesFromChoiceParams({ Metric: "Cases" })
+            expect(decisionMatrix1.selectedRow.grapherId).toEqual(3)
+
+            const decisionMatrix2 = new DecisionMatrix(matrix)
+
+            decisionMatrix2.setValuesFromChoiceParams({
+                "Relative to population": "false",
+            })
+            expect(decisionMatrix2.selectedRow.grapherId).toEqual(6)
+        })
```

Comment:
Is it worth adding a test where the metric exists, but the rest of the options don't?

E.g.  `7,Deaths,Biweekly,false` and the test case:

```ts
const decisionMatrix3 = new DecisionMatrix(matrix)

decisionMatrix3.setValuesFromChoiceParams({ "Metric": "Deaths" })
expect(decisionMatrix3.selectedRow.grapherId).toEqual(7)
```

Reply:
Hm actually the test cases below cover missing options, I don't think this is necessary